src/lib.ts

```diff
@@ -92,9 +96,64 @@ export function advCost(whichTest: number): number {
   }
 }
 
+function statTurnsSaved(whichTest: number): number {
```

Comment:
There is no need for this function to exist, because
(1) We can simply return `60 - advCost(whichtest)` which parses the test page itself to get the most accurate reading, and
(2) If we aren't stopping for the user to manually decide, we would immediately see the number of turns the test took in the next few lines of the log

Reply:
The idea is to return negative numbers if the test is overcapped, so that it's easier to see how overcapped is it. I didn't add the same to non-stat tests because in those cases it's easier to see (and more rare to overcap).